### DISCUSSION TITLE
jm

### DISCUSSION THREAD

**Discussion by xhnsw** [share121/inter-knot#5123] [教程]
分享jm
https://jmcomicgo.me/
![mmexport1726590302727](https://github.com/user-attachments/assets/e7467ec4-5704-4f48-ac37-e51bdb06e27e)

**Comment by JiuXiat** [comment]
小伙子，你很牛啊，在绳网搞色情，还在公共场合传播色情内容
举报了！
小伙子，绳之以法吧！

**Reply by iuhjk** [reply]
说实话都能上github了，真的有人找不到jm吗<img src="https://bbs-static.miyoushe.com/static/2023/08/23/1a7dbc3ba70c173be44e5063a9d15b24_282780687228259316.png" alt="菲米尼-冒泡" width="75" >

**Comment by Yizakl** [comment]
666往教程放
